connectors/src/connectors/slack/index.ts

```diff
@@ -644,6 +668,19 @@ export async function setSlackConfig(
         return slackConfig.disableBot();
       }
     }
+    case "autoReadChannelPattern": {
+      const slackConfig = await SlackConfigurationResource.fetchByConnectorId(
+        connectorId
+      );
+      if (!slackConfig) {
+        return new Err(
+          new Error(
+            `Slack configuration not found for connector ${connectorId}`
+          )
+        );
+      }
+      return slackConfig.setAutoReadChannelPattern(configValue);
```

Comment:
Here you should probably add a comment for empty string and do:
```suggestion
      return slackConfig.setAutoReadChannelPattern(configValue || null);
```